src/main/java/com/onelogin/sdk/conn/Client.java

```diff
@@ -132,6 +132,12 @@ public Client() throws IOException, Error {
 		this(1000);
 	}
 
+    public Client(String clientID, String clientSecret, String region) {
```

Comment:
This file has also wrong format

Reply:
I'm not sure what do you mean. Is there a document\eclipse formatting file that I can import in Eclipse to easily conform to your formatting style ?

Reply:
If you see this file:
https://github.com/MGelbana-Incorta/onelogin-java-sdk/blob/a962cbf89b458adb10393bbd666983eff46e5757/src/main/java/com/onelogin/sdk/conn/Client.java#L135
It has wrong format

Reply:
I see. I pushed a commit that uses tabs instead of spaces. I applied the same formatting to all test classes.